README.md

```diff
@@ -1,12 +1,27 @@
-# Survey
+# ODK-X Survey
 
 This project is __*actively maintained*__
 
+**ODK-X Survey** is an Android application for performing data collection in the ODK-X framework.
+It operates similarly to ODK Collect, but is based on HTML, CSS, and Javascript rather than native Android, and is more flexible in its presentation and execution
```

Comment:
We were trying to remove all references to ODK in the readme to help with the ODK-X/ODK confusion when people search online. Therefore, I think it's best not to do the comparison in the readme.